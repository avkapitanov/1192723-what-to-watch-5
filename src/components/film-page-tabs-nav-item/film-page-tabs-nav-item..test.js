import React from "react";
import renderer from "react-test-renderer";
import FilmPageTabsNavItem from "./film-page-tabs-nav-item";

const tabName = `Overview`;

describe(`FilmPageTabsNavItem component render correctly`, () => {
  it(`Active tab`, () => {
    const tree = renderer.create(
        <FilmPageTabsNavItem
          isActive={true}
          tabName={tabName}
          onActiveTabChange={() => {}}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Inactive tab`, () => {
    const tree = renderer.create(
        <FilmPageTabsNavItem
          isActive={false}
          tabName={tabName}
          onActiveTabChange={() => {}}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});

