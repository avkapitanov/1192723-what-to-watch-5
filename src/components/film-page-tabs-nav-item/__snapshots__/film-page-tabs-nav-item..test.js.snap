// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmPageTabsNavItem component render correctly Active tab 1`] = `
<li
  className="movie-nav__item movie-nav__item--active"
>
  <a
    className="movie-nav__link"
    href="#"
    onClick={[Function]}
  >
    Overview
  </a>
</li>
`;

exports[`FilmPageTabsNavItem component render correctly Inactive tab 1`] = `
<li
  className="movie-nav__item "
>
  <a
    className="movie-nav__link"
    href="#"
    onClick={[Function]}
  >
    Overview
  </a>
</li>
`;
